### PR TITLE
Update demo card link to point to monorepo

### PR DESCRIPTION
**Pull request by huayunh**
Changes proposed in this Pull Request:
- Updated React demo card link to point to the newly published monorepos (or storybook)

### CODE REVIEW

**Comment by huayunh** [comment]
one minute, let me merge the conflicts

**Comment by jeffvg** [comment]
can someone verify this....
https://snack.expo.io/@git/github.com/pxblue/loading-states@reactnative?preview=true&platform=ios

loading

File: src/docs/patterns/login.mdx [hunk 1]
```diff
@@ -56,3 +56,7 @@ To register for an account, a new user must supply their e-mail address. After e
 After the email address is verified, users can continue the registration process by choosing a password. You should display password requirements (if any) near this field.
 
 > **NOTE:** The minimum requirements for registration are an email address, confirmation code, and password. You can also collect additional parameters as necessary for your application (e.g., Name, Phone Number, etc.).
+
+# Developer
+
+PX Blue has an npm package, [@pxblue/react-native-auth-workflow](https://www.npmjs.com/package/@pxblue/react-native-auth-workflow), for React Native. It includes pre-build implementations of the screens for Login, Forgot Password, Contact Information, Self-Registration, Registration By Invitation, and Change Password.
```
Comment (joebochill) [inline src/docs/patterns/login.mdx:62]:
```suggestion
PX Blue has an npm package ([@pxblue/react-native-auth-workflow](https://www.npmjs.com/package/@pxblue/react-native-auth-workflow)) for React Native. It includes pre-built implementations of the screens for Login, Forgot Password, Contact Information, Self-Registration, Registration By Invitation, and Change Password.
```

**Comment by joebochill** [comment]
@jeffvg same for me, but when I update to the latest expo version it loads fine

**Comment by jeffvg** [comment]
yeah seems loading state is using 34.0.1

**Comment by huayunh** [comment]
IMG_0023
My phone

**Pull request merged**